native-code/ios/index.md

```diff
@@ -85,12 +81,22 @@ with the new arguments.
 
 ~~~~~ bash
 # debug build for 64-bit iOS
-gn gen out/ios_64 --args='target_os="ios" target_cpu="arm64" is_component_build=false'
```

Comment:
This is great, since it will always be false on iOS anyway: https://cs.chromium.org/chromium/src/build/config/BUILDCONFIG.gn?rcl=40fdca287f3492e18cd4e9ff513c3b846a1d8089&l=164